spec/dd-agent_spec.rb

```diff
@@ -268,11 +268,12 @@ def set_env_var(name, value)
           :file_cache_path => 'C:/chef/cache'
         ) do |node|
           node.normal['datadog'] = {
-            'agent6' => false,
+            'agent6' => true,
             'api_key' => 'somethingnotnil',
             'agent6_version' => {
               'debian' => '1:6.9.0-1',
               'rhel' => '6.9.0-1',
+              'windows' => '6.9.0'
```

Comment:
nit, you can ignore: trailing commas are nice :)

 ```suggestion
               'windows' => '6.9.0',
```